ocaml/sdk-gen/component-test/jsonrpc-server/server.py

```diff
@@ -0,0 +1,62 @@
+"""
+Module Name:
+    jsonrpc_server
+Description:
+    This module provides a simple JSON-RPC server implementation using aiohttp.
+"""
+
+import json
+import os
+
+from aiohttp import web
+
+
+def load_json_files():
+    """
+    Load all JSON files from the 'spec' directory and merge their contents
+    into a dictionary.
+
+    Returns:
+        dict: A dictionary containing the merged contents of all JSON files.
+    """
+    data = {}
+    for filename in os.listdir("spec/"):
+        if filename.endswith(".json"):
+            with open(f"spec/{filename}", "r", encoding="utf-8") as f:
+                data.update(json.load(f))
+    return data
+
+
+async def handle(request):
+    """
+    Handle incoming requests and execute methods based on the test ID provided
+    in the request headers.
+
+    Args:
+        request (aiohttp.web.Request): The incoming HTTP request.
+
+    Returns:
+        aiohttp.web.Response: The HTTP response containing the JSON-RPC result.
+    """
+    spec = load_json_files()
+    test_id = request.headers.get("User-Agent")
```

Comment:
Another custom header could be defined for this purpose.
The "User-Agent" can be a test target as well.